jdaviz/app.py

```diff
@@ -2171,9 +2193,11 @@ def _create_viewer_item(self, viewer, vid=None, name=None, reference=None):
             'config': self.config,  # give viewer access to app config/layout
             'data_open': False,
             'collapse': True,
-            'reference': reference}
+            'reference': reference,
+            'linked_by_wcs': linked_by_wcs,
```

Comment:
I don't love this living in such a general place, but unfortunately we don't have the flexibility to put it elsewhere right now, so we'll have to live with it for now